docs_src/configuration.md

```diff
@@ -1,29 +1,13 @@
 # Configuration
 
-This page lists all of the relevant configuration parameters for each service in the Automated Checkout reference implementation.
+This page lists all of the relevant configuration parameters for each service in the Automated Vending reference implementation.
 
 !!!info
     Note that this document likely does not cover EdgeX-specific configuration parameters. Application and device service SDK documentation can be found in the [EdgeX Foundry GitHub repositories](https://github.com/edgexfoundry) or in the [official EdgeX documentation here](https://docs.edgexfoundry.org/2.2/).
 
 ## Environment overrides
 
-The simplest way to change one of the configuration values described below is via the use of environment variable overrides in the docker compose file. The value of each configuration item in a service's configuration can be overridden with an environment variable specific to that item. The name of the environment variable is the path to the item in the configuration tree with underscores separating the nodes. The character case of each node in the environment variable name must match that found in the service's configuration. Here are a few examples for the `DriverConfig` section:
-
-```toml
-[DriverConfig]
-  VID = "65535" # 0xFFFF
-  PID = "53"    # 0x0035
-```
-
-```yaml
-Driver_VID: "256" ** Good **
-Driver_PID: "26"  ** Good **
-
-DRIVER_VID: "256" ** BAD **
-driver_pid: "26"  ** BAD **
-```
-
-These overrides are placed in the target service's environment section of the compose file. Here is an example:
+The simplest way to change one of the configuration values described below is via the use of environment variable overrides in the docker compose file. The value of each configuration item in a service's configuration can be overridden with an environment variable specific to that item. The name of the environment variable is the path to the item in the configuration tree with underscores separating the nodes. The character case of each node in the environment variable name must match that found in the service's configuration. These overrides are placed in the target service's environment section of the compose file. Here is an example:
```

Comment:
You removed the `values described below` need so reference/example here.

configuration.yaml
```yaml
Driver:
  VID: "65535" # 0xFFFF
  PID: "53"    # 0x0035
```

compose file
```yaml
DRIVER_VID: "256" 
DRIVER_PID: "26"  
```

Reply:
Right, They are not in the current as this is just an example of how to change configuration via env overrides.